src/Database/Concerns/HasRoles.php

```diff
@@ -30,25 +30,37 @@ public function roles()
     /**
      * Assign the given role to the model.
      *
-     * @param  \Silber\Bouncer\Database\Role|string  $role
+     * @param  \Silber\Bouncer\Database\Role|string|array  $role
      * @return $this
      */
-    public function assign($role)
+    public function assign($roles)
     {
-        (new AssignsRole($role))->to($this);
+        if (is_array($roles)) {
+            foreach ($roles as $role) {
```

Comment:
Isn't this kind of defeating the purpose of wanting to run it all in a single query?